src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -0,0 +1,51 @@
+package edu.oregonstate.mist.locations.frontend.resources
+
+import com.codahale.metrics.annotation.Timed
+import edu.oregonstate.mist.api.AuthenticatedUser
+import io.dropwizard.auth.Auth
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.WebApplicationException
+import javax.ws.rs.client.Client
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+@Path("/locations")
+@Produces(MediaType.APPLICATION_JSON)
+class LocationResource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LocationResource.class)
```

Comment:
i notice that this constant is not used, but i assume you plan to use it in future revisions. before you implement logging by hand, however, i recommend determining to what extent dropwizard will do it for you. most of the web server logging is taken care of automatically, for example.